src/Twilio/Rest/Client.php

```diff
@@ -208,6 +211,7 @@ public function getArg(?string $arg, string $envVar): ?string {
     public function request(string $method, string $uri, array $params = [], array $data = [], array $headers = [], string $username = null, string $password = null, int $timeout = null): \Twilio\Http\Response {
         $username = $username ?: $this->username;
         $password = $password ?: $this->password;
+        getenv('DEBUG_HTTP_TRAFFIC') === 'true' ? $logLevel = 'debug' : $logLevel = $this->getLogLevel();
```

Comment:
Much prefer to flip this so assignment doesn't happen _inside_ the ternary operation.

```
$logLevel = (getenv('DEBUG_HTTP_TRAFFIC') === 'true' ? 'debug' : $this->getLogLevel());
```